Assets/Scripts/ScriptingSystem/NumberContainer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NumberContainer : MonoBehaviour
+{
+    private protected float Value = 0;
```

Comment:
No need to define 0, this is already default if nothing is entered, the compiler will do this for you.